packages/docs/page-config/services/i18n/code/vue-i18n-runtime.ts

```diff
@@ -5,5 +5,5 @@ const { locale, messages } = useI18n();
 const { mergeIntoConfig } = useI18nConfig();
 
 watch(locale, (newLocale) => {
-  mergeIntoConfig(messages[newLocale]["vuestic"]);
+  mergeIntoConfig(messages[newLocale]["anyArbitraryKey"]);
```

Comment:
I think we don't need this anymore, because vuestic watch for locale automatically now. So we can simply remove this demo/requirement.